test/support/install_files/config_test.exs

```diff
@@ -0,0 +1,4 @@
+import Config
```

Comment:
Rename this file to something _not_ ending at `_test.exs` to avoid https://github.com/BeaconCMS/beacon/actions/runs/3960139390/jobs/6783873632